components/eam/src/physics/crm/physpkg.F90

```diff
@@ -1,3033 +1,1829 @@
+  ! Initialize variables in physconst which are not temporally and/or spatially constant
+  !----------------------------------------------------------------------------- 
+  call physconst_init()
 
-    call tracers_init()
+  ! Initialize debugging a physics column
+  call phys_debug_init()
 
-    ! age of air tracers
-    call aoa_tracers_init()
+  call pbuf_initialize(pbuf2d)
 
-    teout_idx = pbuf_get_index( 'TEOUT')
+  !initialize physics update interface routine
+  call physics_update_init()
 
-    ! For adiabatic or ideal physics don't need to initialize any of the
-    ! parameterizations below:
-    if (adiabatic .or. ideal_phys) return
+  ! diag_init makes addfld calls for dyn fields that are output from the physics decomp
+  call diag_init()
 
-    if (nsrest .eq. 0) then
-      call phys_inidat(cam_out, pbuf2d) 
-    end if
-    
-    ! wv_saturation is relatively independent of everything else and
-    ! low level, so init it early. Must at least do this before radiation.
-    call wv_sat_init
-
-    ! CAM3 prescribed aerosols
-    if (cam3_aero_data_on) call cam3_aero_data_init(phys_state)
-
-    ! Initialize rad constituents and their properties
-    call rad_cnst_init()
-    call aer_rad_props_init()
-    call cloud_rad_props_init()
-
-    ! solar irradiance data modules
-    call solar_data_init()
-
-    ! Prognostic chemistry.
-    call chem_init(phys_state,pbuf2d, species_class)
-
-    ! Prescribed tracers
-    call prescribed_ozone_init()
-    call prescribed_ghg_init()
-    call prescribed_aero_init()
-    call aerodep_flx_init()
-    call aircraft_emit_init(phys_state,pbuf2d)
-    !when is_cmip6_volc is true ,cmip6 style volcanic file is read
-    !Initialized to .false. here but it gets its values from prescribed_volcaero_init
-    is_cmip6_volc = .false. 
-    call prescribed_volcaero_init(is_cmip6_volc)
-
-    ! Initialize ocean data
-    if (has_mam_mom) then
-       call init_ocean_data()
-    end if
+  call check_energy_init()
 
-    ! co2 cycle            
-    if (co2_transport()) then
-       call co2_init()
-    end if
+  call tracers_init()
 
-    ! CAM3 prescribed ozone
-    if (cam3_ozone_data_on) call cam3_ozone_data_init(phys_state)
+  ! age of air tracers
+  call aoa_tracers_init()
 
-    call gw_init()
+  teout_idx = pbuf_get_index( 'TEOUT')
 
-    call rayleigh_friction_init()
+  if (nsrest .eq. 0) call phys_inidat(cam_out, pbuf2d) 
+  
+  ! wv_saturation is relatively independent of everything else and
+  ! low level, so init it early. Must at least do this before radiation.
+  call wv_sat_init()
 
-    call pbl_utils_init(gravit, karman, cpair, rair, zvir)
-    if (.not. do_clubb_sgs) call vertical_diffusion_init(pbuf2d)
+  ! Initialize rad constituents and their properties
+  call rad_cnst_init()
+  call aer_rad_props_init()
+  call cloud_rad_props_init()
 
-    if ( waccmx_is('ionosphere') .or. waccmx_is('neutral') ) then
```

Comment:
is MMF not working with WACCM?

Reply:
This has never been tested and I don't understand WACCM well enough to speculate on whether it would work. I intentionally removed all WACCM support with the idea that it should be carefully added back if we really want to support it. That seems like a long ways away, so no sense leaving untested functionality lying around.